pkg/beacon/relay/chain/chain.go

```diff
@@ -38,6 +40,9 @@ type RelayEntryInterface interface {
 	// supposed to submit a relay entry, did not deliver it within a specified
 	// time frame (relayEntryTimeout) counted in blocks.
 	ReportRelayEntryTimeout() error
+	// CurrentEntryStartBlock returns the starting block of the
+	// current relay entry.
+	CurrentEntryStartBlock() (*big.Int, error)
```

Comment:
I asked @dimpar to rename it to `CurrentRequestStartBlock` in #1698. I missed to rename it in the contract earlier and we are having a mess in the naming. Everything but this one is `currentRequest*`.